Inseok_45/services/postService.js

```diff
@@ -0,0 +1,66 @@
+ const createPostDao = require('../models/createPostDao');
+const getPostDao = require('../models/getPostDao');
+const getPosts = require('../models/getPostDao');
+const deletePostDao = require('../models/deletePostDao');
+const editPostDao = require('../models/editPostDao')
+
+const createPosts = async ( userId, postImage, postParagraph ) => {
+    const post = await createPostDao.post(
+        userId,
+        postImage,
+        postParagraph
+    );
+
+    return post;
+};
+
+const getAllPosts = async ( ) => {
+    try {
+        const allPost = await getPostDao.getAllPosts();
+        return allPost;
+    } catch (err) {
+        const error = new Error('Could not get post');
+        error.statusCode = 404;
+        throw error;
+    }
+};
+
+const getUserPosts = async ( userId ) => {
+    try {
+        const userPost = await getPostDao.getUserPosts(
+            userId
+        );
+        return userPost;
```

Comment:
```suggestion
        return getPostDao.getUserPosts(
            userId
        );
```
별다른 로직이 있지 않으므로, 변수 선언 및 비동기처리(await) 없이 곧바로 return 해주는것이 더 깔끔해보입니다~ :)

다른 곳에도 적용될 수 있어보입니다!